tedana/tests/test_model_kundu_metrics.py

```diff
@@ -0,0 +1,37 @@
+"""
+Tests for tedana.model.fit
+"""
+
+import pytest
+import numpy as np
+import pandas as pd
+
+from tedana.model import fit
+
+
+def test_smoke_kundu_metrics():
+    """
+    Smoke test for kundu metrics function
```

Comment:
To be clear, this is just to make sure that nothing is so wrong as to be on fire, not a test specific to the multi-echo or imaging domain, right? If that's the case I have a slight objection to calling it smoke test, in that at some point we would really like the function to just be a full test, so that later we'll have to rename it as it's filled out.

Reply:
We won't rename this test- we'll either delete it or add a separate one for meaningful testing.

Reply:
Sorry, didn't realize that was the plan. That's cool, then, and most of my other comments don't apply.